man/adoc/bpftrace.adoc

```diff
@@ -137,6 +137,12 @@ u:lib.so:"fn(char const*)" { printf("arg0:%s\n", str(arg0));}
 *--info*::
   Print detailed information about features supported by the kernel and the bpftrace build.
 
+*--disable-feature* _feature,feature,..._::
+  Disable detected features, valid values are::
+
+  *uprobe_multi* to disable uprobe_multi link +
+  *kprobe_multi* to disable kprobe_multi link
```

Comment:
How about `--no-feature` instead? It's shorter and is more similar to gcc. I plan on adding `--feature` at some point for the AOT stuff (soon I promise...)

Reply:
sure, will change